src/mono/sample/wasm/browser/Wasm.Browser.CJS.Sample.csproj

```diff
@@ -1,19 +1,19 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration>Debug</Configuration>
     <WasmCopyAppZipToHelixTestDir Condition="'$(ArchiveTests)' == 'true'">true</WasmCopyAppZipToHelixTestDir>
-    <WasmMainJSPath>main.js</WasmMainJSPath>
+    <WasmMainJSPath>main.cjs</WasmMainJSPath>
```

Comment:
why?

Reply:
no strong reason, just mental consistency. Shall I revert back  to `.js` ?

Reply:
I reverted.